bindings/java/src/main/com/apple/foundationdb/Database.java

```diff
@@ -49,6 +49,14 @@
 	 * been created.
 	 */
 	CompletableFuture<Void> allocateTenant(byte[] tenantName);
+
+	/**
+	 * Creates a new tenant in the cluster. Convenience method that accepts and packs a Tuple.
```

Comment:
Just for a little more clarity about what we're packing:

```suggestion
	 * Creates a new tenant in the cluster. This is a convenience method that generates the tenant name by packing a {@code Tuple}.
```

A similar change would apply to each of the functions.